### PR TITLE
docs: add summary to breaking changes

### PR DESCRIPTION
**Pull request by joshka**

### CODE REVIEW

File: BREAKING-CHANGES.md [hunk 1]
```diff
@@ -6,6 +6,31 @@ github with a [breaking change] label.
...
+  - MSRV is now 1.67.0
+- [v0.22.0](#v0220)
+  - serde representation of `Borders` and `Modifiers` has changed
+- [v0.21.0](#v0210)
+  - MSRV is 1.65.0
+  - `terminal::ViewPort` is now an enum
+  - "".as_ref() must be annotated to implement `Into<Text<'a>>`
+  - `Marker::Block` renders as a block char instead of a bar char
+- [v0.20.0](#v0200)
+  - MSRV is no 1.63.0
```
Comment (kdheepak) [inline BREAKING-CHANGES.md:31]:
```suggestion
  - MSRV is now 1.63.0
```

**Review by kdheepak — APPROVED** [review]

File: BREAKING-CHANGES.md (hunk 1 above)
Comment (joshka) [reply BREAKING-CHANGES.md]: https://github.com/ratatui-org/ratatui/commit/c52b0d2d81af2c5a162eaf0a94816aab4cd6dc72

File: BREAKING-CHANGES.md [hunk 2]
```diff
@@ -6,6 +6,31 @@ github with a [breaking change] label.
...
+
+- [Unreleased (v0.24.0)](#unreleased-0240)
+  - `ScrollbarState`: `position`, `content_length`, and `viewport_content_length` are now `usize`
+  - `BorderType`: `line_symbols` is now `border_symbols` and returns `symbols::border::set`
+  - `Frame<'a, B: Backend>` is now `Frame<'a>`
+  - `Stylize` shorthands for `String` now consume the value and return `Span<'static>`
+  - `Spans` is removed
+- [v0.23.0](#v0230)
+  - `Scrollbar`: `track_symbol` now takes `Option<&str>`
+  - `Scrollbar`: symbos moved to `symbols` module
```
Comment (kdheepak) [inline BREAKING-CHANGES.md:21]:
```suggestion
  - `Scrollbar`: symbols moved to `symbols` module
```

File: BREAKING-CHANGES.md [hunk 3]
```diff
@@ -6,6 +6,31 @@ github with a [breaking change] label.
...
+  - MSRV is now 1.67.0
+- [v0.22.0](#v0220)
+  - serde representation of `Borders` and `Modifiers` has changed
+- [v0.21.0](#v0210)
+  - MSRV is now 1.65.0
+  - `terminal::ViewPort` is now an enum
+  - "".as_ref() must be annotated to implement `Into<Text<'a>>`
```
Comment (kdheepak) [inline BREAKING-CHANGES.md:28]:
```suggestion
  - `"".as_ref()` must be annotated to implement `Into<Text<'a>>`
```

**Review by orhun — APPROVED** [review]

**Pull request merged**